### PR TITLE
STAC tutorials: Basic search and plotting GEDI02_B points

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
This is just a draft PR for comment. I think we will keep the STAC build of the documentation separate for now until we decide how we will move forward with the STAC catalog from the broader team (including the users)

### CODE REVIEW

**Comment by sumantjha** [comment]
@abarciauskas-bgse I used the ReviewNB to add a couple comments/suggestions for the notebook. They are not blockers though, just suggestions. You can address them or skip them. Apart from those, it all looks good to me.

**Review by sumantjha — APPROVED** [review]
Please check out my comments in ReviewNB. Just a couple suggestions.

File: docs/source/search/searching_the_stac_catalog.ipynb [hunk 1]
```diff
@@ -0,0 +1,277 @@
+{
```
Comment (spa0002) [inline docs/source/search/searching_the_stac_catalog.ipynb:1]: <p>Recommend adding the function name <code>get_all_collections</code> to the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (spa0002) [inline docs/source/search/searching_the_stac_catalog.ipynb:1]: <p>Perhaps provide information about STAC Items.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: docs/source/tutorials/GEDI/GEDI02_B_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,2615 @@
+{
```
Comment (spa0002) [inline docs/source/tutorials/GEDI/GEDI02_B_tutorial.ipynb:1]: <p>Recommend listing folium as a prerequisite.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Review by spa0002 — APPROVED** [review]

File: docs/source/search/searching_the_stac_catalog.ipynb (hunk 1 above)
Comment (spa0002) [reply docs/source/search/searching_the_stac_catalog.ipynb:1]: <p>The sentence may not matter, I was just thinking that <code>get_all_collections</code> might be good to mention before the code block that uses it. </p>

**Pull request merged**